test/components/reconciliation-reports/reconciliation-report.js

```diff
@@ -212,13 +230,19 @@ test('correctly renders the heading', function (t) {
 test('report with error triggers error message', function (t) {
   const match = { params: { reconciliationReportName: 'exampleReportWithError' } };
 
-  const dispatch = () => {};
+  const dispatch = () => { };
+
+  const granules = () => { };
+
+  const workflowOptions = () => {[]};
```

Comment:
Might be nice to define all these variables outside of the tests so that they can be reused